frame/support/procedural/src/pallet/expand/storage.rs

```diff
@@ -50,7 +66,7 @@ pub fn process_generics(def: &mut Def) {
 			_ => unreachable!("Checked by def"),
 		};
 
```

Comment:
the uniqueness of the attribute is no longer checked by the rust syntax, I think we should check it.
We should iterate on all prefix and ensure they are unique.